src/plugins/template/src/compiled_model.cpp

```diff
@@ -179,8 +179,12 @@ void ov::template_plugin::CompiledModel::export_model(std::ostream& model_stream
     model_stream.write(reinterpret_cast<char*>(&dataSize), sizeof(dataSize));
     model_stream.write(m_model.c_str(), dataSize);
 
-    dataSize = static_cast<std::uint64_t>(m_constants.size());
-    model_stream.write(reinterpret_cast<char*>(&dataSize), sizeof(dataSize));
-    model_stream.write(reinterpret_cast<char*>(&m_constants[0]), dataSize);
+    if (m_cfg.weights_path.empty()) {
```

Comment:
is m_cfg.weights_path used for both import and export?
can we use weightless import but export with weights?

Reply:
I will use `CacheMode` property to export as weightless blob